### DISCUSSION TITLE
Future of SSBH Editor

### DISCUSSION THREAD

**Discussion by ScanMountGoat** [ScanMountGoat/ssbh_editor#171] [Announcements]
I plan to be only minimally involved in the Ultimate modding community going forward. This also applies to the development of tooling. SSBH Editor is not dead. Maintenance for SSBH Editor as well as related projects and tools like ssbh_wgpu or Ultimate Tex will still continue. This will mostly just consist of updates to dependencies and small bug fixes. There are no additional features planned that aren't already documented on GitHub. I've outlined some guidelines for how to help with continued development.

- Please reach out if you have any issues or suggestions. The discussion forums are a great way to get help from the community.
- If you want to add a feature yourself, please start a discussion so I can outline what code changes will be involved and estimate the difficulty. SSBH Editor is typically more beginner friendly to work with than ssbh_wgpu or ssbh_lib.
- If you need to ask a game rendering related question, please ask on https://github.com/ScanMountGoat/Smush-Material-Research/discussions.
- I don't use the tools anymore other than the testing done prior to releases. If you've discovered a bug that isn't already an issue, please make a new GitHub issue so I'm aware of it and can hopefully fix it for the next release.
- GitHub is the best way to communicate with me going forward. I'm not part of any Ultimate modding discords and don't check gamebanana regularly. Creating a discussion or issue is also the only reliable way for me to see your comments.

**Comment by ThatNintendoNerd** [comment]
Thank you for all your contributions! Your tools have been some of the most important to this community, and we are all grateful for your time and effort.
